### PR TITLE
:memo: Docs refactor

### PR DESCRIPTION
**Pull request by acco**

### CODE REVIEW

**Comment by acco** [comment]
* **#570** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/570?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/570?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: docs/how-sequin-works.mdx [hunk 1]
```diff
@@ -1,25 +1,220 @@
 ---
 title: 'How Sequin works'
```
Comment (acco) [inline docs/how-sequin-works.mdx:2]: IGNORE THIS DOC, it is mid-flight

File: docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
+---
+title: "How to stream Postgres to a webhook endpoint"
+sidebarTitle: "Stream to webhook endpoint"
+description: "Receive Postgres changes as webhooks in real-time"
+---
+
+This guide shows you how to use Sequin to set up real-time streaming of database changes from Postgres to your webhook endpoints.
```
Comment (thisisgoldman) [inline docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx:7]: I wonder if we need to anchor on CDC more to echo the value prop:

```suggestion
This guide shows you how to use Sequin to set up Postgres CDC to stream database changes to your webhook endpoints.
```

File: docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+A simple strategy is to generate a secure token and require it in the `Authorization` header of inbound requests. You'll supply this token to Sequin by adding it as a header for your HTTP endpoint.
+
+Here's how to generate a secure token with `openssl`:
+
+```bash
+openssl rand -base64 32
+```
+
+### (Optional) Create a Change Capture Pipeline
```
Comment (thisisgoldman) [inline docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx:54]: The big value prop here is retention on changes. So maybe the headline is "(Optional) Enable retention for changes" or something like that. And then the section highlight when you might want to do that and how (using the change capture pipeline)

File: docs/introduction.mdx [hunk 4]
```diff
@@ -41,56 +41,38 @@ Sequin guarantees exactly-once processing. That means you can use it to:
...
+- Redis Streams
+- Webhooks (HTTP POST)
+- Sequin Stream (HTTP Pull/GET)
+- More coming soon! (NATS, EventHubs, Pub/Sub, SNS, Kinesis, RabbitMQ, RedPanda)
 
 ## Killer features
 
-- **Never miss a row or change:** Sequin ensures all database changes are delivered and processed exactly once by consumers.
-- **SQL-based routing:** Filter and route messages to consumers using SQL `where` conditions.
-- **Replays:** Rewind consumers to any row on your table. Or republish select messages that match a SQL query.
```
Comment (thisisgoldman) [inline docs/introduction.mdx:50]: Should we keep rewind as a valuable option?

File: docs/introduction.mdx
Comment (thisisgoldman) [inline docs/introduction.mdx:1]: I really like how tight this is!

File: docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+With Postgres data streaming to your application, you can trigger workflows, keep services in sync, [build audit logs](/how-to/create-audit-logs), [maintain caches](/how-to/maintain-caches), and more.
+
+By the end of this how-to, you'll have a configured HTTP endpoint receiving database changes.
+
+## Prerequisites
+
+This guide assumes you:
+
+1. [Already have Sequin installed](/quickstart/webhooks)
```
Comment (RTLS) [inline docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx:17]: is this the right link for installation?

Comment (acco) [reply docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx:17]: @RTLS I'm thinking for every how-to, the pre-requisite will be a quickstart for the relevant sink

File: docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+  <Step title="Specify filters">
+    If you selected changes, in "Records to process", you can indicate whether you want to receive `insert`, `update`, and/or `delete` changes.
+
+    You can also specify [SQL filters](/reference/filters) to narrow down the webhooks you want to receive. For example, if you only want to receive webhooks for `subscriptions` that currently have an `mrr` greater than $100, you can add a filter on `mrr > 100`.
+  </Step>
+
+  <Step title="Specify backfill">
+    You can optionally indicate if you want your webhook endpoint to receive a [backfill](reference/backfills) of all or a portion of the table's existing data. Backfills are useful if you want your webhook endpoint to process historical data. For example, if you're materializing a cache, you might want to warm it with existing rows.
+
+    You can backfill at any time. If you don't want to backfill, toggle "Backfill" off.
```
Comment (RTLS) [inline docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx:99]: `backfill` terminology seems correct to me

File: docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+
+1. Make some changes in your source table.
+2. Verify that the count of messages for your sink increases in the Sequin web console.
+3. Verify receipt of messages in your webhook endpoint logs.
+
+If messages don't seem to be flowing:
+
+1. Click the "Messages" tab to view the state of messages for your sink.
+2. Click any failed message.
+3. Check the delivery logs for error details, including the response from your webhook endpoint.
```
Comment (RTLS) [inline docs/how-to/stream-postgres-to-a-webhook-endpoint.mdx:154]: yep we need to add responses here!

**Pull request merged**